docs/resources/azapi_resource_action.md

```diff
@@ -84,6 +84,8 @@ The following arguments are supported:
 
 * `body` - (Optional) A JSON object that contains the request body.
 
+* `locks` - (Optional) A list of ARM resource IDs which are used to avoid modify azapi resources at the same time.
```

Comment:
This looks like not necessarily need to be a resource ID?

Reply:
Yes, but I want to align this description with `azapi_resource` and users might get confused if it says any strings are acceptable.